go/client/chat_common.go

```diff
@@ -20,8 +39,26 @@ func CheckUserOrTeamName(ctx context.Context, g *libkb.GlobalContext, name strin
 		ret := keybase1.UserOrTeamResult_TEAM
 		return &ret, nil
 	}
+	ase, okNet := err.(libkb.AppStatusError)
+	_, okNotFound = err.(teams.TeamDoesNotExistError)
+
+	if !(okNet && keybase1.StatusCode(ase.Code) == keybase1.StatusCode_SCTeamNotFound) && !okNotFound && !strings.HasSuffix(err.Error(), "does not exist") && !strings.HasPrefix(err.Error(), "invalid team name") {
```

Comment:
hacks necessary because functions aren't returning error structs, just messages that need to be manually checked